### ISSUE TITLE
Strange error with scipy on win11

### ISSUE THREAD

**Issue by davidebaltieri31** [scipy/scipy#17861]
I'm on Windows 11, running python 3.10.9 (the error happens also in python 3.10.8)
I create a new venv, and install scipy 1.10 (and numpy 1.24.1) with pip install --upgrade --force-reinstall scipy
in the venv I run python and a simple import (from scipy.linalg import _fblas), everything works.
Restart python and run again the same import and this time I get an importerror (ImportError: DLL load failed while importing _fblas: %1 is not a valid Win32 application.)

```PS I:\> python -m venv testvenv
PS I:\> .\testvenv\Scripts\activate
(testvenv) PS I:\> pip install --upgrade --force-reinstall scipy
Collecting scipy
  Using cached scipy-1.10.0-cp310-cp310-win_amd64.whl (42.5 MB)
Collecting numpy<1.27.0,>=1.19.5
  Using cached numpy-1.24.1-cp310-cp310-win_amd64.whl (14.8 MB)
Installing collected packages: numpy, scipy
Successfully installed numpy-1.24.1 scipy-1.10.0
(testvenv) PS I:\> python
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from scipy.linalg import _fblas
>>> ^Z

(testvenv) PS I:\> python
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from scipy.linalg import _fblas
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "I:\testvenv\lib\site-packages\scipy\linalg\__init__.py", line 197, in <module>
    from ._misc import *
  File "I:\testvenv\lib\site-packages\scipy\linalg\_misc.py", line 3, in <module>
    from .blas import get_blas_funcs
  File "I:\testvenv\lib\site-packages\scipy\linalg\blas.py", line 213, in <module>
    from scipy.linalg import _fblas
ImportError: DLL load failed while importing _fblas: %1 is not a valid Win32 application.
>>> ^Z

(testvenv) PS I:\>```

**Comment by rkern** [comment]
I'm just spitballing here: Do you have an antivirus program running? If so, can you check its logs to see if it did anything to any files under `I:\testvenv` in the time between the two runs? If you `pip install scipy`, then wait for 5 minutes before testing anything, do you still see the same pattern?

**Comment by davidebaltieri31** [comment]
No antivirus, apart from the default one in windows and it didn't log anything in protection history. I tried disabling it anyway but I got the same pattern. Tried waiting but I still got the same pattern too. First time it works fine, second time it gives the ImportError. I also tried taking a snapshot of the env dir soon after a reinstall of scipy and after the error appear and did a byte-by-byte comparison of every file, no changes.

**Comment by ilayn** [comment]
Could you please also verify that after installing SciPy don't run Python and wait for a while just in case Bitlocker is doing something funny and you are catching up. Very strange indeed especially if it is byte-by-byte equivalent.

**Comment by tupui** [comment]
Thank you for reporting @davidebaltieri31. A quick flyby note about your usage of SciPy's API. Note that end users are not supposed to use anything private. i.e. you should not import anything directly from a submodule marked with an underscore or use a function marked with an underscore. We don't provide any guarantee (functions may be removed, API changed, etc.) for the private API.

**Comment by davidebaltieri31** [comment]
Hi, I'm not actively using private apis, I found the problem using a library (torch_fidelity) that depends on scipy and found out that the problem occurred during import of _fblas, so I created a simplified pattern that recreate the problem.

Well, anyway it looks like the problem solved itself. Or rather windows solved the problem for me with an update of the Asus motherboard HAL drivers. Have absolutely no idea how the two could be correlated...
